### PR TITLE
[CLI][Project] Add option to save secrets

### PR DESCRIPTION
**Pull request by GiladShapira94**
Until now when a user set secrets in the CLI (mlrun project function) it added the secrets to the proj object but does not save it in the project itself (does not present in the UI under secrets) causing an issue when a user will try to run for example scheduling workflow that needs those secret it will not found any secrets because they are saved only in the proj object.

I kept the original code lines and add proj,.set_secrets to take the secrets and save them in the project for later use

### CODE REVIEW

**Comment by AlonMaor14** [comment]
Generally looks good but why not do it the other way around?
By default save the secrets

File: mlrun/__main__.py [hunk 1]
```diff
@@ -988,6 +988,11 @@ def logs(uid, project, offset, db, watch):
     is_flag=True,
     help="Overwrite a schedule when submitting a new one with the same name.",
 )
+@click.option(
+    "--save-secrets",
+    is_flag=True,
+    help="Save secrets as project secrets.",
```
Comment (AlonMaor14) [inline mlrun/__main__.py:994]:
```suggestion
    help="Save secrets as project secrets in the db",
```

**Review by AlonMaor14 — COMMENTED** [review]
Generally LGTM, minor comment.

**Review by AlonMaor14 — APPROVED** [review]

File: mlrun/__main__.py [hunk 2]
```diff
@@ -1045,7 +1051,13 @@ def project(
         proj.spec.params["commit_id"] = commit
     if secrets:
         secrets = line2keylist(secrets, "kind", "source")
-        proj._secrets = SecretsStore.from_list(secrets)
+        secret_store = SecretsStore.from_list(secrets)
+        # Allow the user to run a workflow with secrets on runtime, without using or storing k8s secrets.
+        # if the user wants to run a schedule workflow or to use those secrets in other runs he will need to save
+        # those secrets in k8s and use the --save-secret flag
```
Comment (jillnogold) [inline mlrun/__main__.py:1057]: Used to run a workflow with secrets in runtime, without using or storing k8s secrets.
To run a scheduled workflow or to use those secrets in other runs, save
the secrets in k8s and use the --save-secret flag

Comment (GiladShapira94) [reply mlrun/__main__.py]: Fix, thanks

**Review by Tankilevitch — APPROVED** [review]

**Pull request merged**